table/sst_file_writer_collectors.h

```diff
@@ -33,6 +33,14 @@ class SstFileWriterPropertiesCollector : public IntTblPropCollector {
     return Status::OK();
   }
 
+  virtual void BlockAdd(uint64_t /* blockRawBytes */,
+                        uint64_t /* blockCompressedBytesFast */,
+                        uint64_t /* blockCompressedBytesSlow */) override {
+    // Intentionally left blank. No interest in collecting stats for
+    // blocks.
```

Comment:
`SstFileWriter` is another good place for compression sampling, may be done in a separate PR.